test/debug_info/entry_argument_1.f90

```diff
@@ -0,0 +1,20 @@
+!RUN: %flang -g -S -emit-llvm %s -o - | FileCheck %s
+
+!CHECK: [[N1:![0-9]+]] = distinct !DISubprogram
+!CHECK: !DILocalVariable(arg: 1, scope: [[N1]]
+!CHECK: !DILocalVariable(arg: 2, scope: [[N1]]
```

Comment:
I suggest changing the patterns to ensure that these two `DILocalVariable` nodes have the `DIFlagArtificial` flag.

Reply:
I was mistaking some other variables for the artificial arguments. What you have is fine.